.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -103,9 +103,7 @@ labels: 'issue: bug'
 
 (Write your steps here:)
 
-1.
-2.
-3.
+1. 2. 3.
```

Comment:
This came from GitHub's template, the new format is causing strange output:
![image](https://user-images.githubusercontent.com/5043083/61858995-ebdb6c80-aecf-11e9-8e49-6276633fd837.png)

It might be best to revert this line?

Reply:
I didn't even realize I changed that. Must be VSCode's Markdown formatter. I'll revert it.

Reply:
So it's actually prettier that does this in the pre-commit hook. I had to edit it directly in GitHub to get around that. 😞

Reply:
Well... strange. But at least you got it sorted.

I think the original had a space between each line - perhaps that was why.